propriedade/tests/views/test_propriedade.py

```diff
@@ -111,3 +112,21 @@ def test_nao_cria_propriedade_cep_invalido(self):
             'CEP deve possuir 8 digítos numéricos.',
             response.json()['cep']
         )
+
+    def test_lista_propriedade_de_um_produtor_autenticado(self):
+        self.client.credentials(HTTP_AUTHORIZATION='Bearer ' + self.get_header_credencial(self.produtor.usuario))
+
+        propriedade.make(produtor=self.produtor, tecnico=self.tecnico)
```

Comment:
adicionar uma propriedade para outro produtor. Ex.:
```suggestion
        propriedade.make(produtor=self.produtor, tecnico=self.tecnico)
        propriedade.make(produtor=produtor.make(), tecnico=self.tecnico)
```